src/exceptions/property_permission.php

```diff
@@ -50,9 +50,11 @@ class ezcBasePropertyPermissionException extends ezcBaseException
      */
     function __construct( $name, $mode )
     {
-        parent::__construct( "The property '{$name}' is " .
+        parent::__construct(
```

Comment:
This is a widespread use in Zeta to have code of the form `( "foo", "bar", \n\n..... \n, .. "baz" );` and it would be split up into the first argument going on the next line after the call with the current standard.

I tried getting it to do differently, but its not configurable in phpcs. Without this rule, we could not automatically enforce the space before and after arguments as that is part of the same phpcs rule.

Reply:
FWIW, I think this is a good change.